### ISSUE TITLE
[BUG] Password cannot be changed in a fresh installation with defaults

### ISSUE THREAD

**Issue by AndrewBedscastle** [immauss/openvas#250]
**Describe the bug**
 Password cannot be changed in a fresh installation with defaults

**To Reproduce**
1. Include how you started the container. 
  Docker Compose start

```
openvas:
      environment:
        - "RELAYHOST=172.17.0.1"
        - "SMTPPORT=25"
        - "REDISDBS=512" # number of Redis DBs to use
        - "QUIET=false"  # dump feed sync noise to /dev/null
        - "NEWDB=false"  # only use this for creating a blank DB 
        - "SKIPSYNC=true" # Skips the feed sync on startup.
        - "RESTORE=false"  # This probably not be used from compose... see docs.
        - "DEBUG=false"  # This will cause the container to stop and not actually start gvmd
        - "HTTPS=false"  # wether to use HTTPS or not
        - "GMP=9390"    # to enable see docs
      volumes:
        - ./data/openvas:/data
      container_name: openvas
      image: immauss/openvas:latest
      network_mode: service:another_container
      labels:
      - "com.centurylinklabs.watchtower.enable=true" 
```

docker compose up 

After Logging in with admin / admin went to Administration -> User -> Admin and tried to change password.

Error occurs in docker compose logs -f log

> error opening '/etc/gvm/pwpolicy.conf': No such file or directory

Password is not changed; admin / admin combination remains
**This is unsafe**, as the UI shows no errors

2. When did the issue occur? 

Fresh start with docker compose file above

**Expected behavior**
Password should have been  changed

**Environment (please complete the following information):**
 - OS: Ubuntu 23.10
 - Memory available to OS: 16G
 - Container environment used with version: Docker CE

**Additional context**
Workaround

docker exec -it openvas /bin/bash
cd /etc/gvm
touch pwpolicy.conf
exit

**Comment by immauss** [comment]
Thanks. I'll have this straight in the next release which should be out in a day or so.

**Comment by immauss** [comment]
This is resolved in the current latest and 22.4.40. 

Thank you !

**Comment by AndrewBedscastle** [comment]
Wow that was fast. Thank you very much and thank you for the work you put in this project!
